Dockerfile

```diff
@@ -1,17 +1,20 @@
-ARG CP_KAFKA_CONNECT_BASE_VERSION=6.0.1
+ARG CP_KAFKA_CONNECT_BASE_VERSION=7.2.2.amd64
 
 FROM confluentinc/cp-kafka-connect-base:$CP_KAFKA_CONNECT_BASE_VERSION
 
 ENV COMPONENT=kafka-connect
+ENV CONNECT_PLUGIN_PATH=/usr/share/java,/usr/share/confluent-hub-components,/usr/share/filestream-connectors
 
-ARG CONFLUENTINC_JDBC_CONNECTOR_VERSION=10.0.1
-ARG CONFLUENTINC_HDFS_CONNECTOR_VERSION=10.0.0
-ARG CONFLUENTINC_ELASTICSEARCH_CONNECTOR_VERSION=11.0.0
-ARG CONFLUENTINC_S3_CONNECTOR_VERSION=5.5.3
-ARG CONFLUENTINC_AVRO_CONVERTER_CONNECTOR_VERSION=5.5.3
+ARG CONFLUENTINC_JDBC_CONNECTOR_VERSION=10.5.4
+ARG CONFLUENTINC_HDFS_CONNECTOR_VERSION=10.1.13
+ARG CONFLUENTINC_ELASTICSEARCH_CONNECTOR_VERSION=14.0.2
+ARG CONFLUENTINC_S3_CONNECTOR_VERSION=10.2.2
+ARG CONFLUENTINC_AVRO_CONVERTER_CONNECTOR_VERSION=7.2.2
+ARG JCUSTENBORDER_SPOOL_CONNECTOR_VERSION=2.0.64
 
-RUN confluent-hub install --no-prompt confluentinc/kafka-connect-jdbc:$CONFLUENTINC_JDBC_CONNECTOR_VERSION \
-    && confluent-hub install --no-prompt confluentinc/kafka-connect-hdfs:$CONFLUENTINC_HDFS_CONNECTOR_VERSION \
-    && confluent-hub install --no-prompt confluentinc/kafka-connect-elasticsearch:$CONFLUENTINC_ELASTICSEARCH_CONNECTOR_VERSION \
-    && confluent-hub install --no-prompt confluentinc/kafka-connect-s3:$CONFLUENTINC_S3_CONNECTOR_VERSION \
-    && confluent-hub install --no-prompt confluentinc/kafka-connect-avro-converter:$CONFLUENTINC_AVRO_CONVERTER_CONNECTOR_VERSION
\ No newline at end of file
+RUN confluent-hub install --no-prompt confluentinc/kafka-connect-hdfs:$CONFLUENTINC_HDFS_CONNECTOR_VERSION \
+&& confluent-hub install --no-prompt confluentinc/kafka-connect-jdbc:$CONFLUENTINC_JDBC_CONNECTOR_VERSION \
+&& confluent-hub install --no-prompt jcustenborder/kafka-connect-spooldir:$JCUSTENBORDER_SPOOL_CONNECTOR_VERSION \
+&& confluent-hub install --no-prompt confluentinc/kafka-connect-elasticsearch:$CONFLUENTINC_ELASTICSEARCH_CONNECTOR_VERSION \
+&& confluent-hub install --no-prompt confluentinc/kafka-connect-s3:$CONFLUENTINC_S3_CONNECTOR_VERSION \
+&& confluent-hub install --no-prompt confluentinc/kafka-connect-avro-converter:$CONFLUENTINC_AVRO_CONVERTER_CONNECTOR_VERSION
```

Comment:
**Optional**: This **MAY** be prettier terminated with a new line for terminal processing.
```suggestion
&& confluent-hub install --no-prompt confluentinc/kafka-connect-avro-converter:$CONFLUENTINC_AVRO_CONVERTER_CONNECTOR_VERSION

```